Support parquet bulk load for P8

theneelshah opened the pull request:
Issue #751:

Description of changes:

- Handle parquet format for bulk load in P8.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

adityaramesh12 commented:
Modified the description message.
Can you verify/validate if incremental load with neptune database is possible as well? In that case, we would need to add an explicit condition to only permit parquet load for Neptune Analytics.

theneelshah commented:
Parquet format is only supported for incremental loads, which are exclusively used with Neptune Analytics. Bulk loads (used with Neptune DB) do not support the parquet format. This distinction is handled in the load magic function when processing the format parameter.

theneelshah commented:
@adityaramesh12: Fixed the bug where edgeOnlyLoad parameter is passed for all bulk load formats. Currently, edgeOnlyLoad is only supported for CSV or Parquet format loads. Also, as per the previous comment, bulk load doesn't support parquet format loads. This behaviour might be changed and hence I have added a condition for parquet format for edgeOnlyLoad param.

adityaramesh12 commented on src/graph_notebook/magics/graph_magic.py:
```diff
@@ -2491,7 +2511,7 @@ def on_button_clicked(b):
 
                     kwargs.update(bulk_load_kwargs)
-
+                print(kwargs)
```
nit: can be removed?

theneelshah replied:
Sorry, removed.
